sql/core/src/test/resources/sql-tests/inputs/postgreSQL/window_part2.sql

```diff
@@ -99,6 +111,7 @@ FROM tenk1 WHERE unique1 < 10;
 -- nth_value(salary, 1) over(order by salary range between 1000 preceding and 1000 following),
 -- salary from empsalary;
 
+-- [SPARK-30734] AnalysisException that window RangeFrame not match RowFrame
 select last(salary) over(order by salary range between 1000 preceding and 1000 following),
```

Comment:
Sorry if I wasn't clear. I think this one still should comment this case out since we don't support. This is a positive case in PostgreSQL, and looks odds to convert this one alone to a negative test cases.

Reply:
OK